src/network/protocol/messages/addr.zig

```diff
@@ -0,0 +1,180 @@
+const std = @import("std");
+const protocol = @import("../lib.zig");
+
+const Endian = std.builtin.Endian;
+const Sha256 = std.crypto.hash.sha2.Sha256;
+
+const CompactSizeUint = @import("bitcoin-primitives").types.CompatSizeUint;
+
+pub const NetworkIPAddr = struct {
```

Comment:
part of it already exists under src/network/protocol/NetworkAddress.zig

please use it

Reply:
You can reorder the fields if you want

Reply:
so you have a struct which is time + NetworkAddress